cargo-apk/src/apk.rs

```diff
@@ -47,49 +47,64 @@ impl<'a> ApkBuilder<'a> {
             Artifact::Root(name) => format!("rust.{}", name.replace("-", "_")),
             Artifact::Example(name) => format!("rust.example.{}", name.replace("-", "_")),
         };
-        let package_label = self
-            .manifest
-            .metadata
-            .apk_label
-            .as_deref()
-            .unwrap_or(artifact.name())
-            .to_string();
 
-        let config = Config {
+        // Set default Android manifest values
+        let mut manifest = self.manifest.android_manifest.clone();
```

Comment:
Isn't it cleaner to fill this in directly after loading the file in `fn from_subcommand`?